### ISSUE TITLE
[RFC] RIP-82: Improve Knowledge Management

### ISSUE THREAD

**Issue by uschtwill** [raid-guild/RIPs#82]
# [RFC] RIP-82: Improve Knowledge Management

### Project Submitted By

flip, govinda, HHH, uschtwill

## Summary

Improve our knowledge management by consolidating the Guild’s knowledge into an easy to consume, self-service format that allows newcomers and members alike to inform and orient themselves. This will empower them to contribute and take action without the help of others.

To achieve this goal we propose two measures:

- Document all there is to know about the inner workings of the Guild in a single HackMD document (a landing page or index of sorts), in direct sub-documents of that document, or by linking out to other existing HackMD documents or standalone resources (like the Handbook).<br/><br/>To showcase our vision of what this could look like and to serve as an MVP, we created **[a knowledge garden](https://hackmd.io/7dhrZJ5qQ72tffmJ4sCpdQ?view)**.<br/><br/>
- Work to establish a culture of documenting collaboratively, pro-actively and comprehensively.

We believe that this cultural work will be necessary in order for this RIP to have long-lasting effect. Documentation will only stay fresh and up-to-date if it is owned collectively. In fact for this reason the heroic effort of a small group documenting everything by themselves may even be counter-productive to the documentations longevity.

## Why should we do this?

As a new person coming into the Guild it is almost impossible to understand what is going on by yourself. There is some knowledge transfer happening during the cohort process itself, but apart from that, knowledge is mostly implicit or scattered across dozens of (unconnected!) HackMD documents.

For a new person to get up to speed, to find the necessary resources and become able to contribute, they have to show tremendous initiative and ask around a lot on Discord. We also suspect this to be one of the main reasons for the high attrition rates we can observe in the cohorts. Being blind sucks.

At the same time the Guild is at a critical point in its journey: We have dozens of applicants per month wanting to join and applying to the cohort process and with ~140 members, the Guild is about to soar past [Dunbar’s Number](https://en.wikipedia.org/wiki/Dunbar%27s_number).

We will need to formalize our structure and processes to continue to grow, and as such this is also true for our knowledge management. The current informal way of managing our knowledge is starting to hold us back and stiffle our ability to scale.

Futhermore recent literature on DAOs seems to identify documentation as one of the most critical things for DAOs to get right. Here then are some of our members not having sufficiently high-quality documentation and proposing solutions  themselves:

- [RaidGuild’s ShillShop Systemization RIP](https://hackmd.io/l9pgahNhTu-Wl1Ym8EKzJw)
- [Raid Guild ops-a look inside](https://hackmd.io/@HHH-HMD/Bk6omxo3t)
- https://github.com/raid-guild/RIPs/issues/80
- https://github.com/raid-guild/RIPs/issues/68
- https://github.com/raid-guild/RIPs/issues/52
- https://github.com/raid-guild/RIPs/issues/41
- https://github.com/raid-guild/RIPs/issues/38

**tl;dr: We need excellent and comprehensive documentation - now.**

## Anything else you'd like to add?

We would like to address a number of thoughts and objections.

### "Come on, it's not that bad."

We understand that it may seem like that for people, who have been with the Guild for some time already. It seems it's actually in the nature of the problem, that the longer you have been with the Guild, the less you can see this issue. When you know your way around and when you were there when decisions were made, processes shaped and tools set up - there is no problem.

But from the perspective of someone who just finished a cohort, the implementation of this RIP would add tremendous value.

### "We have the Handbook, why do we need something else?"

The Handbook is a very useful resource, both to get a first impression of the Guild and also for some of the more involved (on-chain) processes. 

At the same time, its [commit history](https://github.com/raid-guild/handbook/commits/master) seems to suggest that it is also a rather slow-moving piece of documentation. This is okay for external and representative kind of documentation like the Handbook, but this RIP aims to create a living and breathing documentation that will change fast and often. For example, meeting minutes, figma workspaces, interviews, new projects, etc., while ill-suited for the handbook, would be more naturally documented in an easy to edit documentation system.

The Handbook's git workflow may be be an impediment to this as well, being both unconducive to spontaneous changes and a bit scary to non-techies. Making it effortless for everybody to contribute to this is crucial, see next point.

### "There have been many efforts like this before. Most of the time, these sort of documents become stale and useless. Why will this effort be different?"

We understand that documentation like this is only useful when the whole community collaborates and shares a culture of regularly updating it to reflect reality. 

For this to happen it is actually counterproductive if on person or a small group go on a documentation frenzy and heroically create a polished page. For a culture of shared knowledge management to emerge it is necessary for everybody to contribute and own the documentation. It is much easier to embrace and nurture something that you yourself have helped build.

This RIP acknowledges this fact and is much more about coordination, generating buy-in and creating a culture of shared knowledge management than it is about writing down things per se.

Of course, voting for and funding this RIP in itself is already a shared commitment.

### "Why HackMD? If we are going to do this, aren't there better tools for this?"

There are many tools that could be used for the purpose of this RIP: Notion, CMS-based wiki systems, clarity.so, etc. [All of these have different trade-offs](https://creators.mirror.xyz/qTeJ-VeKfuijliliIZHmbFYzcIZXx3lJdbORVdPWDUk) and even our own TravisWyche is [pondering how to solve this question in a Web3 way](https://hackmd.io/@TravisWyche/HJ5xh0HhF).

We propose to do this in HackMD for two reasons: 

a) Time is of the essence. We should solve this issue NOW and sholdn't wait for a lengthy tool-evaluation process. HackMD using Markdown also ensures that what we create will be easy to migrate. Once we do settle on a tool for the long-term, the migration will be fairly straight forward.

b) HackMD is very entrenched in the Guild culturally, virtually everybody is used to working with it. With collaboration and culture being the most critical sucess factor for this RIP, using the existing tool with the lowest barrier to participation makes the most sense. Also with most of the existing documentation being on HackMD, using the same tool makes it easy to link it together.

### "I would like to give feedback on this proposal!"

Wonderful! You can do so by either:

- Commenting on this issue
- Visiting us in #internal-guild-wiki on Disord
- Posting on #rip-discussion on Discord
- DMing us on Discord
- You can append the 🪴 emoji to your username on Discord for a while to signal you are generally in favour. :)

## Raid Party Skills Needed

- Scribes / Content Creators
- Knowledge Management Experts
- Collaboration/Coordination Experts

## Cost 

We distinguish between two types of work:

- Tier 1 (indirect):
    - Culture work
    - Nudging/coaching members
    - Project management
- Tier 2 (direct):
    - Writing documentation
    - Layouting of Knowledge Garden 
    - Structuring documentation

We suggest the following rates:
- Tier 1: $120/h
- Tier 2: $90/h

While this proposal hinges on the cultural aspect of knowledge management, there will also be (and already has been) a considerable amount of Tier 2 work. Hence, we propose a 50/50 split of the total time into 36 hours each.

The budget thus is:

| Type | Time | Cost/h | Total Cost |
| -------- | -------- | -------- |-------- |
| Tier 1     | 36h     | $120/h     |  $4,320 |
| Tier 2     | 36h     | $90/h     |  $3,240 |
| **Total**     | 72h     | -     |  **$7,560** |

**Disclaimer:** 6h of T2 budget will be used to compensate *govinda* retroactively, and so will 4h of T1 and T2 hours each for *uschtwill*. This will fund:

-  The current [Knowledge Garden skeleton](https://hackmd.io/7dhrZJ5qQ72tffmJ4sCpdQ?view#Brand) 
- The gathering of and transferrence of all the centralised information contained on Govinda's RG [Notion links and resources list](https://raidguild.notion.site/Welcome-to-Raid-Guild-b019b430590c4b65a504704f257ac83a)
-  The [RG Internal Ops Q&A documentation](https://hackmd.io/@HHH-HMD/Bk6omxo3t) with OGs
-  The scoping of this project as conducted on Discord and summarised [here](https://raidguild.notion.site/Summary-of-documentation-discussion-67adff6f2bcf4fb4995b685d95c8edac)

## Timeline

We are planning with the following timeline:

- Until mid-February: Get RIP passed
- Mid-February - Beginning of March: Preparation (allow ETHDenver delegates to recover)
- Raider Roundup on 1st of March: Kick off concerted effort/campaign lasting ~ 1 month

## Deliverables

- "Knowledge Garden" living knowledge repository
- 36h of collaborative work focused on impacting culture

We are aware of the fact that as a deliverable, 36h of culture work are hard to measure. At the same time we are absolutely certain, that the RIP needs it to produce something of long-lasting value.

**Comment by uschtwill** [comment]
Other RIP contributors have most recent markdown. Will probably be re-opened by them.
